applications/mne_analyze/plugins/averaging/averaging.h

```diff
@@ -206,12 +206,6 @@ class AVERAGINGSHARED_EXPORT Averaging : public ANSHAREDLIB::IPlugin
      */
     void computeAverage();
 
-    //=========================================================================================================
-    /**
-     * Saves state of 'use averaging / use stim' checkboxes based on gui
-     */
-    void onCheckBoxStateChanged();
-
 //    //=========================================================================================================
```

Comment:
do we still need this commented out function?

Reply:
we do not, will remove